### DISCUSSION TITLE
Undervolting: any REAL benefit?

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#663] [General]
Hi there, I have a question to people who use the CPU undervolting feature:

Did you notice any REAL benefits in doing it? Maybe with some before/after tests and data?

I'll probabily try the feature myself but I am just curious about other's experiences.

I undervolted my Steam Deck for a total of -60mv and the benefits are within measurements error, so I can't imagine it doing much on this notebook...

**Comment by ke1ne** [comment]
@LoreAPC better temperatures during "hard" tasks

**Comment by LoreAPC** [comment]
How many degrees and with how much undervolting?

**Reply by ke1ne** [reply]
> How many degrees and with how much undervolting?

Didn't test G-Helper undervolt with gaming YET, but with another tool, previously, I got kind of 5-10 degrees of profit. Which was nice because of less fan noise and heat on keyboard.

**Reply by LoreAPC** [reply]
Five degrees is a LOT!!! How much mv did you undervolt?

**Reply by ke1ne** [reply]
> Five degrees is a LOT!!! How much mv did you undervolt?

It was kind of 9 months ago, I can't remember. Just googled something like "Ryzen Controller settings for G15". I also was surprized, btw.

p.s. I will definitely try to test again, but later.
